notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -297,15 +297,8 @@ public Page updateNote(Page note, PageUpdateType type) throws WikiException {
     updateNote(note);
     invalidateCache(note);
 
-    if (PageUpdateType.EDIT_PAGE_CONTENT.equals(type) || PageUpdateType.EDIT_PAGE_CONTENT_AND_TITLE.equals(type)) {
-      try {
-        listenerService.broadcast("exo.wiki.edit", wikiService, note);
-      } catch (Exception e) {
-        log.error("Error while broadcasting wiki edition event", e);
-      }
-    }
-
     Page updatedPage = getNoteById(note.getId());
+    Utils.broadcast(listenerService, "note.updated", note.getLastUpdater(), updatedPage);
```

Comment:
We should broadcast listeners as well in `public Page updateNote(Page note)` to ensure information consistency & coherence even in batch method.